deployment/envgen2/test/src/templates/add_binding_esdl_1.diff

```diff
@@ -0,0 +1,25 @@
+734a735,744
```

Comment:
Not convinced this file should be here. Or a lot of other files in this directory, come to that.

Reply:
all of test directory is to run test for envgen2. The *.diff files are for checking the results. The test.sh generate *.diff which will be compared with *.diff under templates/.  I can put template environment.xml files. But there will be lots of them and each is about 50-100KB.
test directory will not be packaged. Otherwise not sure where I should put them.